libs/blocks/chart/chart.js

```diff
@@ -116,28 +150,43 @@ export const getChartOptions = (chartType, dataset, colors, unit = '') => {
 };
 
 const init = async (el) => {
+  const parent = el.parentElement;
   const children = el?.querySelectorAll(':scope > div');
   children[0]?.classList.add('chart_title');
   children[1]?.classList.add('chart_subTitle');
   children[3]?.classList.add('chart_footnote');
 
-  // ToDo: Replace hardcoded size MWPW-112994
-  const size = LARGE;
+  // Add chart container
+  const container = document.createElement('div');
+  container.className = 'chart_container';
+  container.append(...children);
+  el.appendChild(container);
```

Comment:
Is it really necessary to add another div? It seems like the md adds plenty of divs. But if you do have to inset a new element, you should use semantic html and choose something like `<section>`

Reply:
That's a good point. I have updated to using `<section>` instead.